website/src/models/challenge.js

```diff
@@ -97,7 +97,6 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
   let [challengeTasks, userTasks] = await Q.all([
     // Find original challenge tasks
     Tasks.Task.find({
-      userId: {$exists: false},
```

Comment:
The point was that we want to fetch the challenge tasks and sync them to the user so there's no point in fetching the challenge tasks that are already synced to the users.